packages/react-i18n/src/test/i18n.test.ts

```diff
@@ -1192,151 +1192,173 @@ describe('I18n', () => {
-        );
-      });
+        it('formats a date less than one week ago', () => {
+          const fiveDaysInMilliseconds = 5 * 24 * 60 * 60 * 1000;
+          const today = new Date('2012-12-20T00:00:00-00:00');
+          const lessThanOneWeekAgo = new Date(
+            today.getTime() - fiveDaysInMilliseconds,
+          );
+          clock.mock(today);
+          const i18n = new I18n(defaultTranslations, {
+            ...defaultDetails,
+            timezone,
+          });
 
-      it('formats a date less than one week ago', () => {
-        const fiveDaysInMilliseconds = 5 * 24 * 60 * 60 * 1000;
-        const today = new Date('2012-12-20T00:00:00-00:00');
-        const lessThanOneWeekAgo = new Date(
-          today.getTime() - fiveDaysInMilliseconds,
-        );
-        clock.mock(today);
-        const i18n = new I18n(defaultTranslations, {
-          ...defaultDetails,
-          timezone,
+          i18n.formatDate(lessThanOneWeekAgo, {
+            style: DateStyle.Humanize,
+          });
+          expect(translate).toHaveBeenCalledWith(
+            'date.humanize.lessThanOneWeekAgo',
+            {
+              pseudotranslate: false,
+              replacements: {weekday: 'Saturday', time: '11:00 a.m.'},
+            },
+            defaultTranslations,
+            i18n.locale,
+          );
         });
 
-        i18n.formatDate(lessThanOneWeekAgo, {
-          style: DateStyle.Humanize,
+        it('formats a date less than one year ago', () => {
+          // Offset from 2012-12-20 to 2012-07-20
+          const fiveMonthsInMilliseconds =
+            (30 + 31 + 30 + 31 + 31) * 24 * 60 * 60 * 1000;
+
+          const today = new Date('2012-12-20T00:00:00-00:00');
+          const lessThanOneYearAgo = new Date(
+            today.getTime() - fiveMonthsInMilliseconds,
+          );
+          clock.mock(today);
+          const i18n = new I18n(defaultTranslations, {
+            ...defaultDetails,
+            timezone,
+          });
+
+          i18n.formatDate(lessThanOneYearAgo, {
+            style: DateStyle.Humanize,
+          });
+          expect(translate).toHaveBeenCalledWith(
+            'date.humanize.lessThanOneYearAgo',
+            {
+              pseudotranslate: false,
+              replacements: {date: 'Jul. 20', time: '10:00 a.m.'},
+            },
+            defaultTranslations,
+            i18n.locale,
+          );
         });
-        expect(translate).toHaveBeenCalledWith(
-          'date.humanize.lessThanOneWeekAgo',
-          {
-            pseudotranslate: false,
-            replacements: {weekday: 'Saturday', time: '11:00 a.m.'},
-          },
-          defaultTranslations,
-          i18n.locale,
-        );
       });
 
-      it('formats a date less than one year ago', () => {
-        // Offset from 2012-12-20 to 2012-07-20
-        const fiveMonthsInMilliseconds =
-          (30 + 31 + 30 + 31 + 31) * 24 * 60 * 60 * 1000;
+      describe('future dates', () => {
```

Comment:
Do you want to add a test for today but in the future?

Reply:
Yep! I can definitely add that test 👍